cms/envs/common.py

```diff
@@ -738,23 +769,9 @@
 # The name of the require.js script used by your project, relative to REQUIRE_BASE_URL.
 REQUIRE_JS = "js/vendor/requiresjs/require.js"
 
-# A dictionary of standalone modules to build with almond.js.
```

Comment:
Almond seems to be a stripped down runtime for module loading. As far as I can tell, we don't use it, and I'm not aware of a time when we did.

https://github.com/requirejs/almond